src/core/engine/app_engine_processor.py

```diff
@@ -1,18 +1,26 @@
 # thread manager
 
-from src.core.engine.app_engine_processor_context import \
-    AppEngineProcessorContext
+from src.core.engine.app_engine_processor_context import AppEngineProcessorContext
 from src.core.engine.app_processor_config import AppProcessorConfig
 from src.core.engine.processors.kitchen_simulator import KitchenSimulator
+from src.core.order_manager import OrderManager
 
 
 class AppEngineProcessor:
     def __init__(self):
-        kitchen_simulator_config = AppProcessorConfig("kitchen_simulator_test", 0.2)
+        kitchen_simulator_config = AppProcessorConfig(
+            id="kitchen_simulator_test", interval=0.2, order_manager=OrderManager()
```

Comment:
could you set `on_start` here instead? like
```
AppProcessorConfig(
            id="kitchen_simulator_test", interval=0.2, order_manager=OrderManager(), on_start=kitchen_simulator_module.initialize
)
```

Reply:
done.